Doc/library/enum.rst

```diff
@@ -925,3 +925,17 @@ Notes
        >>> from enum import IntEnum
        >>> class MyIntEnum(IntEnum):
        ...     __str__ = IntEnum.__str__
+
+   Additionally, it's possible to end up shadowing a built-in function from
+   a derived baseclass such as :meth:`str.title` or :meth:`int.bit_count`,
```

Comment:
Let's add one more example here. It feels like 2 is not enough :)

Reply:
Added another example in afde363e70c626697e04e141359413caff0d708e